packages/client-sdk-nodejs/src/preview-storage-client.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  AbstractStorageClient,
+  IStorageControlClient,
+  IStorageDataClient,
+} from '@gomomento/sdk-core/dist/src/internal/clients';
+import {IStorageClient} from '@gomomento/sdk-core/dist/src/clients/IStorageClient';
+import {StorageClientProps} from './storage-client-props';
+import {StorageClientPropsWithConfig} from './internal/storage-client-props-with-config';
+import {StorageControlClient} from './internal/storage-control-client';
+import {StorageDataClient} from './internal/storage-data-client';
+import {StorageConfiguration} from './config/storage-configuration';
+import {StorageConfigurations} from './index';
+
+export class PreviewStorageClient
```

Comment:
Should have some docstrings here?

This is what I wrote in for the go PR: https://github.com/momentohq/client-sdk-go/pull/413#discussion_r1643445922